reactioncommerce/reaction-next-starterkit
# feat: display order surcharges in UI

Must be merged at the same time as of after https://github.com/reactioncommerce/reaction/pull/4829

Impact: **minor**
Type: **feature**

## Issue
We've added the ability to have surcharges attached to an order. This PR displays the surcharges in the UI. Depends on the updates in https://github.com/reactioncommerce/reaction/pull/4829

## Solution
Retrieve surcharge data, and display in Starterkit UI.

## Breaking changes
None

## Testing
1. Add product to cart that has surcharges attached (see https://github.com/reactioncommerce/reaction/pull/4829 for details on adding surcharges)
1. See that surcharges are displayed in checkout, and that you can complete an order
